### DISCUSSION TITLE
Action is successful, but the website doesn't appear.

### DISCUSSION THREAD

**Discussion by NeonGooRoo** [JamesIves/github-pages-deploy-action#1541] [Q&A]
It may be a dumb question but I am lost. My repo is https://github.com/NeonGooRoo/NeonGooRu and after pushing the build works fine, but when I open the link there is no website.

The build is all fine in the gh-pages branch, but it doesn't appear on the website https://neongoo.ru/ or https://neongooroo.github.io/NeonGooRu/

What am I missing?

**Comment by JamesIves** [comment]
You likely need to go into your repo settings > pages and change the branch to `gh-pages`.

**Reply by NeonGooRoo** [reply]
Oh, so I don't need to use deploy option in GitHub pages, that's why. Sorry, thanks
